docs/deployment.md

```diff
@@ -11,54 +11,41 @@ All members of the Apply development team are able to deploy into any of the env
 To speed up the release pipeline, logic has been introduced into the pipeline code to only run the full ARM template deployment only if the following conditions are met.
 - Any of the pipeline variable groups associated with the deployment have changed since the last successful deployment
 - Any of the following files have changed in the commit associated with the build:
-  - azure-pipelines.yml
-  - azure-pipelines-deploy-template.yml
-  - azure-pipelines-release.yml
+  - azure/pipelines/build.yml
+  - azure/pipelines/release.yml
+  - azure/pipelines/templates/deploy.yml
   - azure/template.json
   - azure/containers.json
 If none of the above conditions are met the pipeline will simply load the new image into the container staging slot ready for swapping.
 
 ## 1. Check what you're deploying
 
-Go to [the Apply Ops Dashboard](https://apply-ops-dashboard.herokuapp.com/) and find the commit you want to deploy.
-
-Make sure to check the diff on GitHub to see if there's anything risky.
+Go to [the Apply Ops Dashboard](https://apply-ops-dashboard.azurewebsites.net) and find the commit you want to deploy 
+and check the diff on GitHub to see if there's anything risky.
 
 You also have to make sure that you're deploying only work that is safe to deploy. It should be either behind a feature flag or product reviewed.
 
-## 2. Tell the team ![](https://cultofthepartyparrot.com/parrots/shipitparrot.gif)
-
-Summarise what you're deploying and tell the team in Slack on the `#twd_apply` channel. Use `:ship_it_parrot:` as required.
-
-## 3. Deploy to staging
+## 2. Deploy to staging
 
-1. Load the [apply-for-teacher-training-releases](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=325&_a=summary) page in Azure DevOps.
-1. Click the blue "Run pipeline" button (sometimes it says "Queue") at the top right of the page which will open the run pipeline menu.
-1. Ensure the branch is set to "master".
-1. Specify the commit
-1. Under the Variables section, make sure only `deploy_staging` is set to true (this should be the default)
-1. Click the Run button to start the deployment.
+Click on the `Deploy` button next to the commit sha under staging section in the dashboard.
+You might be prompted to login to Azure AD and grant permissions for the deployment to being, you won't be prompted to login to Azure again if an active session exists.
+Once the deployment beings this will automatically post the list of PRs being deployed to the `#twd_apply` slack channel.
```

Comment:
```suggestion
Click on the `Deploy` button next to the commit SHA under Staging section in the dashboard.

You might be prompted to login to Azure AD and grant permissions for the deployment. You won't be prompted to login to Azure again if an active session exists.

Once the deployment begins, this will automatically post the list of PRs being deployed to the `#twd_apply` slack channel.
```